ios/browser/api/brave_shields/adblock_filter_list+private.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_BRAVE_SHIELDS_ADBLOCK_FILTER_LIST_PRIVATE_H_
+#define BRAVE_IOS_BROWSER_API_BRAVE_SHIELDS_ADBLOCK_FILTER_LIST_PRIVATE_H_
+
+#import <Foundation/Foundation.h>
+#include "brave/ios/browser/api/brave_shields/adblock_filter_list.h"
+
+namespace adblock {
+class FilterList;
+}  // namespace adblock
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface AdblockFilterList (Private)
+- (instancetype)initWithFilterList:(adblock::FilterList)filterList;
+- (adblock::FilterList)filterList;
+@end
```

Comment:
One thing that I'd been meaning to get to is moving the `FilterList` definition outside of the FFI glue code (`namespace adblock`). It used to be defined in `adblock-rust`, but the C++ definition was never moved after it was removed from the Rust code.

If it helps to organize things better in this PR, it'd be great to move that as well - otherwise no worries and I can get to it as a followup eventually.

Reply:
Discussed in DMs, doesn't really change organization in this PR in particular since our API wrappers are already defined under `brave_shields`, will leave as is for now :)